src/authorization/WebAclAuthorizer.ts

```diff
@@ -24,17 +25,24 @@ import { Authorizer } from './Authorizer';
 export class WebAclAuthorizer extends Authorizer {
   protected readonly logger = getLoggerFor(this);
 
-  private readonly aclManager: AclManager;
+  private readonly aclManager: AuxiliaryManager;
   private readonly resourceStore: ResourceStore;
   private readonly identifierStrategy: IdentifierStrategy;
 
-  public constructor(aclManager: AclManager, resourceStore: ResourceStore, identifierStrategy: IdentifierStrategy) {
+  public constructor(aclManager: AuxiliaryManager, resourceStore: ResourceStore,
+    identifierStrategy: IdentifierStrategy) {
     super();
     this.aclManager = aclManager;
     this.resourceStore = resourceStore;
     this.identifierStrategy = identifierStrategy;
   }
 
+  public async canHandle({ identifier }: AuthorizerArgs): Promise<void> {
+    if (this.aclManager.isAuxiliaryIdentifier(identifier)) {
+      throw new NotImplementedHttpError('WebAclAuthorizer does not support permissions on acl files.');
```

Comment:
```suggestion
      throw new NotImplementedHttpError('WebAclAuthorizer does not support permissions on auxiliary resources.');
```

Reply:
The auxiliary manager it takes as input is one specifically for acl files (which it needs to find the correct acl resource for a permission), so the check here will only check specifically for acl resources and not all auxiliary resources in general. "files" still needs to be changed to "resources" though.